docs/installation/kepler-operator.md

```diff
@@ -1,21 +1,22 @@
 # Kepler Operator on Kind
 
-## Requirements:
+## Requirements
 
 Before you start make sure you have:
 
 - `kubectl` installed
-- Clone the `kepler-operator`[repository](https://github.com/sustainable-computing-io/kepler-operator)  
-- You’ll need a Kubernetes cluster to run against. You can use KIND to get a [local cluster for testing](#run-a-kind-cluster-locally), or run against a remote cluster. Note: Your controller will automatically use the current context in your kubeconfig file (i.e. whatever cluster kubectl cluster-info shows).
-- Signed in as `kubeadmin` or a user with `cluster-admin` role
+- Clone the `kepler-operator` [repository](https://github.com/sustainable-computing-io/kepler-operator)
+- You’ll need a Kubernetes cluster to run against. You can use KIND to get a [local cluster for testing](#run-a-kind-cluster-locally), or run against a remote cluster.
 
+  **Note:** Your controller will automatically use the current context in your kubeconfig file (i.e. whatever cluster kubectl cluster-info shows).
 
+- Sign in as `kubeadmin` or a user with `cluster-admin` role
 
-### Run a kind cluster locally
+## Run a kind cluster locally
 
-``` sh
+```sh
 cd kepler-operator
-make cluster-up CLUSTER_PROVIDER='kind' CI_DEPLOY=true GRAFANA_ENABLE=true
+make cluster-up
```

Comment:
Just `make fresh` should do

Reply:
And we shoudn't require `kc get pods -n monitoring`

Reply:
When I run `make cluster-up`  I see the following
```
 🔆🔆🔆  Ensuring all tools are installed  🔆🔆🔆 
━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
 🔔 installing all tools ...
 🔔 installing controller-gen with version: v0.12.1
go: downloading sigs.k8s.io/controller-tools v0.12.1
go: downloading github.com/spf13/cobra v1.7.0
go: downloading k8s.io/api v0.27.1
go: downloading k8s.io/apimachinery v0.27.1
```
But the command failed on a machine with no pre-installed  `kind` 

```
Starting kind with cluster name "kind"

 ❯ kind create cluster --name=kind -v6 --config=/home/parsingh/go/src/github.com/test-pr/kepler-operator/tmp/local-dev-cluster/tmp/kind/kind.yml

/home/parsingh/go/src/github.com/test-pr/kepler-operator/tmp/local-dev-cluster/lib/utils.sh: line 50: kind: command not found
make: *** [Makefile:120: cluster-up] Error 127
```

Shouldn't the `make` command ensures that we have kind pre-installed as well?
I will open a corresponding issue in the `kepler-operator`